plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/AbstractKuduWithDisabledInferSchemaConnectorTest.java

```diff
@@ -31,7 +36,13 @@
     @Test
     public void testListingOfTableForDefaultSchema()
     {
-        assertQuery("SHOW TABLES FROM default", "VALUES ('customer'), ('nation'), ('orders'), ('region')");
+        // Tests in BaseConnectorTests may race and create tables in the default schema
+        // Assert at least the TPCH tables exist but there may be more
+        List<String> rows = new ArrayList<>(computeActual("SHOW TABLES FROM default").getMaterializedRows())
```

Comment:
The comment left me confused for a while. Would it be better to say `Test methods may run in parallel and create tables in the default schema`?

Also you can use `REQUIRED_TPCH_TABLES` here to make the assertion at-least a bit stronger (since they must always exist since they get created during QueryRunnerCreation).

Reply:
Any reason on wrapping it on top of `ArrayList` instead of using `computeActual("SHOW TABLES FROM default").getMaterializedRows().stream()`